src/api/app/views/webui/users/notifications/_notifications_list.html.haml

```diff
@@ -17,23 +17,26 @@
       .list-group.list-group-flush.mt-3
         - notifications.each do |n|
           - notification = NotificationPresenter.new(n)
-          .list-group-item.d-flex.flex-column.flex-sm-row.justify-content-sm-between
-            .content
-              .text-nowrap
-                %span.badge.badge-secondary= notification.notification_badge
-                %small.text-muted= time_ago_in_words(notification.created_at)
-              = link_to(n.title, notification.link_to_notification_target, class: 'text-word-break-all')
-              %p.text-muted= notification.excerpt
-            - unless params[:type] == 'read'
-              .actions.align-self-end.align-self-sm-start.pl-3.pt-3.pt-sm-0
-                = link_to(my_notification_path(id: notification, type: params[:type], project: params[:project]),
-                        method: :put, class: 'btn btn-sm btn-outline-success px-4', title: 'Mark as "Read"', remote: true) do
+          .list-group-item.px-2.grid-container
+            .notifiable
+              = notification_icon(n.notifiable_type)
+              = notification_link(notification)
+              = request_state(notification)
+
+            .actions.ml-auto.align-self-start
+              - unless params[:type] == 'read'
+                = link_to(my_notification_path(id: notification),
+                          method: :put, class: 'btn btn-sm btn-outline-success px-3', title: 'Mark as "Read"') do
                   %i.fas.fa-check
-            - unless notification.unread?
-              .actions.align-self-end.align-self-sm-start.pl-3.pt-3.pt-sm-0
-                = link_to(my_notification_path(id: notification, type: params[:type], project: params[:project]),
-                        method: :put, class: 'btn btn-sm btn-outline-success px-4', title: 'Mark as "Unread"', remote: true) do
+              - unless notification.unread?
```

Comment:
Can we swap the conditions and make them positive instead of negative?

Reply:
yes, maybe we could refactor it a little bit the `actions` block, because the only differences are the `title` and the `icon`

Reply:
Moved to a helper and refactored, @danidoni